cmd/run_controller/main.go

```diff
@@ -77,7 +78,7 @@ func main() {
 	factory.TektonInformerFactory().Start(stopCh)
 
 	klog.V(2).Infof("Run controller")
```

Comment:
May be you could log the treadiness here. 
e.g. "Run controller (x threads)"

Same for tenant controller.

Reply:
done